.github/workflows/deploy.yml

```diff
@@ -0,0 +1,72 @@
+name: Deploy
+
+on: 
+  push:
+    branches:
+      - main
+    
+jobs:
+  deployment:
+
+    runs-on: ubuntu-latest
+    environment: Production
+    steps:
+    - name: Get current time for Folder
+      uses: josStorer/get-current-time@v2.0.1
+      id: current-time
+      with:
+        format: YYYYMMDDHHMMSS
+    - name: Get current time for Cleanup
+      uses: josStorer/get-current-time@v2.0.1
+      id: current-time-cleanup
+      with:
+        format: YYYYMM
+    - uses: actions/checkout@v1
+    - uses: "shivammathur/setup-php@v2"
+      with:
+        php-version: "8.1"
+    - uses: "ramsey/composer-install@v2"
```

Comment:
I might be missing it, but where do you need PHP on the action?

Reply:
to install dependencies since we rsync the entire thing over.